.github/pull_request_template.md

```diff
@@ -1,3 +1,8 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
```

Comment:
I do not feel like we should add the copyright banner here. Probably caused by the copyright script.
```suggestion
```

Reply:
Thanks, I fix the script and remove the header.